helpers/cryptos/lsk_utils.js

```diff
@@ -0,0 +1,291 @@
+            }
+            return {
+              success: false,
+              error: `Unable to broadcast Tx, it may be dust amount or other error`,
+            };
+          } catch (e) {
+            return {
+              success: false,
+              error: e.toString(),
+            };
+          }
+        }
+        return {
+          success: false,
+          error: `Unable to create Tx`,
+        };
+      } catch (e) {
+        return {
+          success: false,
+          error: e.toString(),
+        };
+      }
+    } catch (e) {
+      log.warn(`Error while sending ${params.value} ${this.token} to ${params.address} with ${fee} ${this.token} fee in send() of ${utils.getModuleName(module.id)} module: ` + e);
+      return {
+        success: false, error: e.toString(),
+      };
+    }
+  }
+
+  /**
+   * Creates a signed tx object and ID
+   * Signed JSON tx object is ready for broadcasting to blockchain network
+   * @override
+   * @param {string} address receiver address in Base32 format
+   * @param {number} amount amount to transfer (coins, not beddows)
+   * @param {number} fee transaction fee (coins, not beddows)
+   * @param {number} nonce transaction nonce
+   * @return {Promise<{tx: Object, txId: string}>}
+   */
+  createTransaction(address = '', amount = 0, fee, nonce, data = '') {
+    const { liskTx } = this._buildTransaction(address, amount, fee, nonce, data);
+    // To use transactions.signTransaction, passPhrase is necessary
+    // So we'll use cryptography.signDataWithPrivateKey
+    const liskTxBytes = transactions.getSigningBytes(this.assetSchema, liskTx);
+    const txSignature = cryptography.signDataWithPrivateKey(
+        Buffer.concat([this.networkIdentifier, liskTxBytes]), this.account.keyPair.secretKey,
+    );
+
+    liskTx.signatures[0] = txSignature;
+    const txId = utils.bytesToHex(cryptography.hash(transactions.getBytes(this.assetSchema, liskTx)));
+
+    // To send Tx to node's core API, we should change data types
+    liskTx.senderPublicKey = utils.bytesToHex(liskTx.senderPublicKey);
+    liskTx.nonce = nonce.toString();
+    liskTx.fee = transactions.convertLSKToBeddows((+fee).toFixed(this.decimals));
+    liskTx.asset.amount = transactions.convertLSKToBeddows((+amount).toFixed(this.decimals));
+    liskTx.asset.recipientAddress = utils.bytesToHex(liskTx.asset.recipientAddress);
+    liskTx.signatures[0] = utils.bytesToHex(txSignature);
+    return Promise.resolve({ tx: liskTx, txId });
+  }
+
+  /**
+   * Returns Tx status and details from the blockchain
+   * @override
+   * @param {String} txid Tx ID to fetch
+   * @param {Boolean} disableLogging Disable logging flag
+   * @return {Object} Formed tx
+   */
+  async getTransaction(txid, disableLogging = false) {
+    return this._getService(`${lskService}/api/v2/transactions/`, { transactionId: txid }).then((result) => {
+      if (typeof result !== 'object') return undefined;
+      if (result && result.data[0]) {
```

Comment:
remove the `if` statement above and just use `typeof result === 'object'` && result.data[0]`